### PR TITLE
Add Hail version and driver image version to metadata

### LINKED ISSUE #26: Log Hail version
As part of the metadata, the analysis runner should also log the driver image version that's used, as well as the Hail deployment version, if possible. We'll need to add corresponding columns in the Airtable base.

### PR DESCRIPTION
**Pull request by illusional**
Closes #26.
Need to check what downstream changes are required before going too far.

Can test the `_get_hail_version()` by running:

```bash
cd server/
# maybe comment out lines 37-38 (secret manager and pubsub as they initialise on file load)
docker build .
docker run --entrypoint /bin/bash -it docker.io/cpg/analysis-runner-server:latest

# inside container
# These are required to load the script, as main.py asserts the values exist.
export GCP_PROJECT="some-random-value"
export DATASET="some-random-value" 
echo "from main import _get_hail_version; import asyncio; print(asyncio.run(_get_hail_version()))" | python
# should print version: `0.2.62-9d69512338f7`
```

### CODE REVIEW

File: server/main.py [hunk 1]
```diff
@@ -9,11 +9,12 @@
 from google.cloud import secretmanager
 from google.cloud import pubsub_v1
 
+from hail import version as get_hail_version
 import hailtop.batch as hb
 from hailtop.config import get_deploy_config
 
 GITHUB_ORG = 'populationgenomics'
-
+HAIL_VERSION = get_hail_version()
```
Comment (lgruen) [inline server/main.py:17]: I actually meant the version of the Hail deployment, but I suppose the server version is also important!

Comment (illusional) [reply server/main.py:17]: Blocked by: https://github.com/populationgenomics/hail/issues/34

**Comment by illusional** [comment]
Was blocked by: https://github.com/populationgenomics/hail/pull/51 But this has been merged.

File: server/main.py [hunk 2]
```diff
@@ -165,6 +175,8 @@ async def index(request):
                 'script': ' '.join(script),
                 'description': params['description'],
                 'output': output_dir,
+                'hail': hail_version,
```
Comment (lgruen) [inline server/main.py:178]: Just to make this less ambiguous:

```suggestion
                'hailVersion': hail_version,
```

File: server/main.py [hunk 3]
```diff
@@ -165,6 +175,8 @@ async def index(request):
...
+                'driver_image': DRIVER_IMAGE,
```
Comment (lgruen) [inline server/main.py:179]: CamelCase in JSON?

```suggestion
                'driverImage': DRIVER_IMAGE,
```

**Review by lgruen — APPROVED** [review]
Please also remember to add the new columns to Airtable.
(Including the ones from my recent change :).)

**Pull request merged**